src/main/java/seedu/address/logic/parser/LabelCommandParser.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.LabelCommand;
+import seedu.address.logic.commands.SelectCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+
+public class LabelCommandParser {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the SelectCommand
+     * and returns an SelectCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public LabelCommand parse(String args) throws ParseException {
+        try {
+            Index index = ParserUtil.parseIndex(args);
+            return new LabelCommand(index);
+        } catch (ParseException pe) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, SelectCommand.MESSAGE_USAGE), pe);
```

Comment:
I think there is a minor change to be made in Line 24 as it is currently showing Select Command Error instead of LabelCommand error but overall good job and it also shows code consistency since overall you followed the structure of other commands which is good 👍 

This is a draft fix based on what I read, I try to avoid fixing it directly for you using Github in case you are fixing it.
`String.format(MESSAGE_INVALID_COMMAND_FORMAT, LabelCommand.MESSAGE_USAGE), pe);